### ISSUE TITLE
Explore improvements to notifications when using a screen reader

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#183567]
During an inclusive design workshop focused on developer tools hosted by design teams at Microsoft, a panel of developers with a variety of disabilities singled out notifications as a pain point when using VS Code.

Screen reader users mentioned a few key areas to be improved:
- How the notification announces itself (could be improved with audio cues as well)
- How to know there were actions/buttons on a notification
- How to reference the notification again when it disappears or is accidentally dismissed 
- Understanding the severity of a notification and if it was something they needed to engage with right away

cc @meganrogge

**Comment by meganrogge** [comment]
cc @jooyoungseo 

> How to reference the notification again when it disappears or is accidentally dismissed

We have this command `Notifications: Show Notifications`, but I'm not sure it's very discoverable

> How to know there were actions/buttons on a notification

We could add a hint to the aria label and an `accessibility.verbosity` setting similar to the recent fix for discovering how to change a keybinding when a row is focused in the keybinding editor. 

> How the notification announces itself (could be improved with audio cues as well)

I believe we alert at the moment, so it should be clear that it has come in. We could consider an audio cue though I think that might be extra noise with little added value.

> Understanding the severity of a notification and if it was something they needed to engage with right away

I think that hearing the notification's message should suffice here and we don't want to interpret the severity for a screen reader user.
